Source/Fluids/WarpXFluidContainer.cpp

```diff
@@ -196,9 +203,21 @@ void WarpXFluidContainer::InitData(int lev, amrex::Box init_box)
                 amrex::Real z = problo[0] + i * dx[0];
 #endif
 
+                // Lorentz transform z (from boosted to lab frame)
+                if (WarpX::gamma_boost > 1._rt){
```

Comment:
It seems that there are two `gamma_boost` here:
- `gamma_boost` declared at line 167
- `WarpX::gamma_boost`

Is there a reason for this? Could we instead remove the first one and only use `WarpX::gamma_boost`/`WarpX::beta_boost` instead?

Reply:
Changed to WarpX::gamma_boost/WarpX::beta_boost in the latest commit.